src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCasePreviewService.java

```diff
@@ -343,14 +345,24 @@ protected String buildHeldBefore(SscsCaseData caseData, String userAuthorisation
         }
         names.add(signedInJudgeName);
 
-        List<JudicialUserBase> panelMembers = caseData.getAdjournment().getPanelMembers();
+        Adjournment adjournment = caseData.getAdjournment();
```

Comment:
To clarify, would this need to be under the feature flag also?

Then again, if this is only called from code which can't be accessed until the adjournment changes are in ccd-defs, then I'd imagine it should be fine.

Reply:
no this wont, the panel members as text are still stored in the adjournment class